integration/monai/examples/spleen_ct_segmentation_local/README.md

```diff
@@ -139,36 +146,40 @@ For details about resource management and consumption, please refer to the [docu
 > To speed up your experimentation, you can reduce the `num_rounds` value in `config_fed_server.json`, e.g. to 5 rounds.
 
 ### 5.1 Experiment tracking with MLflow
-Using MONAI's experiment [tracking feature](https://github.com/Project-MONAI/tutorials/tree/main/experiment_management),
-we can use MLflow to track the model performance on the local clients.
+Experiment tracking with MLflow in version 2.4.0 now uses MetricsExchanger through the `NVFlareStatsHandler` and `MetricsRetriever` components to stream events
+to the FL server to write to MLflow. The `spleen_ct_segmentation_local` job should be configured to automatically 
+
+The `spleen_ct_segmentation_loc_non_agg` job is the previous configuration that uses MONAI's experiment [tracking feature](https://github.com/Project-MONAI/tutorials/tree/main/experiment_management)
+with clients logging to the MLflow tracking server without going through the FL server.
+
+For `spleen_ct_segmentation_loc_non_agg`, an MLflow server is expected, so in a new terminal, start the mlflow server with:
 
-In a new terminal, start the mlflow server:
 ```
 mlflow server
 ```
 
 You can access the MLflow dashboard in your browser using the default tracking uri `http://127.0.0.1:5000`.
+
 Next, submit the job.
 
 ### 5.2 Federated averaging
 
-To run FedAvg using with local provisioning, submit the job using:
+To run FedAvg using with the Job CLI, submit the job with:
+
 ```
-./submit_job.sh job
+nvflare job submit -j jobs/spleen_ct_segmentation_local
 ```
-(Optional) In POC mode, use
+
+You can also submit the job with the script:
 ```
-./submit_job.sh job --poc
+./submit_job.sh jobs/spleen_ct_segmentation_local
 ```
```

Comment:
If the job CLI is good enough, maybe we can remove this submit_job.sh?
@holgerroth what do you think?